engine/src/parser/expression_utils.cpp

```diff
@@ -135,6 +136,7 @@ cudf::type_id get_output_type(operator_type op, cudf::type_id input_left_type) {
 		return cudf::type_id::TIMESTAMP_NANOSECONDS;
 	case operator_type::BLZ_STR_LOWER:
 	case operator_type::BLZ_STR_UPPER:
+        case operator_type::BLZ_STR_INITCAP:
```

Comment:
please  fix spacing so that it matches the rest

Reply:
addressed